core/sr-primitives/src/lib.rs

```diff
@@ -658,6 +687,20 @@ mod tests {
 			assert_eq!($per::from_percent(1) * $num_type::max_value(), $num_type::max_value() / 100);
 			assert_eq!($per::from_percent(0) * $num_type::max_value(), 0);
 
+			// from_rational_approximation
```

Comment:
These tests + their name in macro suggest testing _multiplication_. Maybe separate them or rename it to sth more general?